SyncApp/Views/Configrations/Countries.cshtml

```diff
@@ -15,6 +16,22 @@
         $(document).on('click', 'input[type="checkbox"]', function () {
             $('input[type="checkbox"]').not(this).prop('checked', false);
         });
+        function getTaxFromShopify(countryId) {
+        // Make an AJAX request to Configrations controller action
+        $.ajax({
+            type: "GET",
+            url: "@Url.Action("getTaxFromShopify", "Configrations")",
+            data: { id: countryId },
+            success: function(result) {
+                
+               var taxInput = 'Tax_' + countryId;
+               document.getElementById(taxInput).value = result;
+            },
+            error: function(error) {
```

Comment:
when there is an error nothing will show up to the user, maybe show some alert with a message indicating something went wrong, or the exception message.

Reply:
Done